### ISSUE TITLE
Black theme AMOLED

### ISSUE THREAD

**Issue by AlphaOmega3** [rdoeffinger/Dictionary#43]
Add a black theme "AMOLED" for save the battery

**Comment by rdoeffinger** [comment]
There is a black theme included (at least on my devices, the one called "default" in the settings is black).
If there are specific things wrong with it, please list them.
Though UI work is not the kind of thing I'm any good at, so unless it's very simple changes this will probably have to wait for someone else to volunteer to do it.

**Comment by rancidfrog** [comment]
Dark theme does not apply everywhere:
![](https://i.imgur.com/JY7Ls35.png)
Above is using dark theme, when you click on word it displays with white background, which is not dark theme case
Also, in translated dict. There is dark grey and black mixed, it is not uniform, which is strange:
![](https://i.imgur.com/XUJmyqq.png)

**Comment by fayad** [comment]
Following up the last update on this issue. Dark theme does not apply on description page.
I guess issue is applying the main layout across all sections of the app - makes the app more consistent. Could you please look into this.

**Comment by mysteryoneal** [comment]
I find that the dark blue links are hard to read when using dark theme. I don't remember this being an issue until 1 or 2 updates ago, but it is definitely an issue now. The fact that I use Android's blue light filter may be exacerbating the issue. 

An easy fix is to assign a lighter blue or more higher contrast colour to to link text. 

An even better fix is to allow user defined background, text, and link text colours.

**Comment by rdoeffinger** [comment]
Yes, this was something I missed and it's already fixed in the code.
However Google Play is still processing the last update, so it doesn't make much sense to make a fixed release. I guess if you use F-Droid I could trigger an update for that one fairly easily though.
I don't want to add lots of extra settings for picking colours, I think there's already too many settings that also are in need of some polishing.

**Comment by mysteryoneal** [comment]
Fair enough...keep it simple as is usually best. I'll update through
FDroid, thanks

**Comment by mysteryoneal** [comment]
I've just updated via FDriod. The colour for link text in dark theme is a lighter blue (more legible) on the search page. But link text on individual definition pages is still the hard to read dark blue as it was in current Google Play version.

**Comment by rdoeffinger** [comment]
The colours in both seem identical to me now.
Can you double-check that you are on 5.5.6 (ID 111), e.g. via the "about" page?

**Comment by mysteryoneal** [comment]
It looks like FDroid's repository update didn't complete last time and I
failed to realise. So even after uninstalling/reinstalling to resolve an
FDroid/Google Play app ID conflict... I was still running 5.5.4.

Updated again to 5.5.6. All link text is light blue and QuickDic is perfect
again. Resolved, thanks.
